cmdstanpy/model.py

```diff
@@ -493,6 +493,7 @@ def _do_sample(self, stanfit: StanFit, idx: int) -> None:
         """
         cmd = stanfit.cmds[idx]
         self._logger.info('start chain {}.  '.format(idx + 1))
+        self._logger.debug("sampling: %s", cmd)
```

Comment:
please use new style formatting everywhere.

Reply:
Did this need some setup to work correctly (logging format style?), but I'm not sure if this affects our use of the logging module.

Reply:
I see, according to this:  https://docs.python.org/3/howto/logging-cookbook.html#use-of-alternative-formatting-styles  new style formatting requires config - default is old-style formatting, as above.